src/main/java/Problem1/LinkedListStack.java

```diff
@@ -9,26 +9,25 @@
 
     public LinkedListStack() {
         // homework
+        data = new LinkedList<T>();
     }
 
     @Override
     public boolean push(T val) {
         // homework
-        return false;   // place holder
+        return data.add(val);
     }
 
     @Override
     public T pop() {
         // homework
-        T val = null;   // place holder
-        return val;   // place holder
+        return data.removeLast();
```

Comment:
use First instead of Last. 

this was discussed in many different places including our discussion. for example here https://youtu.be/7O_dRR8fC3w. please pay attention to such information.